src/passes/OptimizeInstructions.cpp

```diff
@@ -395,6 +395,25 @@ struct OptimizeInstructions
           return replaceCurrent(un);
         }
       }
+      {
+        // x < 0 || x > POSITIVE_CONST   ==>   x > POSITIVE_CONST (unsigned
+        // comparison)
```

Comment:
```suggestion
        // x < 0 || x > POSITIVE_CONST   ==>   unsigned(x) > POSITIVE_CONST
```
(this is the convention we have elsewhere in the file to indicate an unsigned comparison)

Reply:
Done